src/tools/compiletest/src/runtest/coverage.rs

```diff
@@ -46,9 +47,9 @@ impl<'test> TestCx<'test> {
             &expected_coverage_dump,
         );
 
-        if coverage_dump_errors > 0 {
+        if coverage_dump_errors.should_error() {
```

Comment:
Nit: this local is named a bit strange, it's actually more like `coverage_dump_compare_outcome`, right?